Highlight mouse buttons in the input map editor icons

Calinou opened the pull request:
This makes visual grepping easier when going through the list of inputs.

The icon designs could be improved, especially for the mouse wheel ones. I know we have some existing designs that highlight a mouse button like `InputEventMouseButton.svg`, but the purpose of those designs is different (it's to describe a certain class).

A similar change could also be applied to gamepad buttons/axes, but we'd need icons for those.

**Testing project:** [test_inputs.zip](https://github.com/user-attachments/files/19694813/test_inputs.zip)

## Preview

![Image](https://github.com/user-attachments/assets/837b0d14-e03d-4ddb-9513-1de01923bc3a)

![Image](https://github.com/user-attachments/assets/a661dac3-0e24-4dba-afc1-9fb85784425c)

Calinou committed:
Highlight mouse buttons in the input map editor icons

Co-authored-by: Pāvels Nadtočajevs <7645683+bruvzg@users.noreply.github.com>

bruvzg commented:
I like the idea, but wheel icons are barely readable, I would add a cutout around indication to make it more visible (maybe even bigger cutout than I added):
<img width="228" alt="Screenshot 2025-04-10 at 20 54 38" src="https://github.com/user-attachments/assets/377fb3d7-6329-49ce-b02b-64107cbfe210" />

[Archive.zip](https://github.com/user-attachments/files/19689652/Archive.zip)

Calinou commented:
> I like the idea, but wheel icons are barely readable, I would add a cutout around indication to make it more visible (maybe even bigger cutout than I added):

Thanks for the icons! I've added them to the PR and added you as a co-author :slightly_smiling_face:

Mickeon commented:
I am a bit iffy on the mouse wheel icons, still. They're an improvement, sure, but are still somewhat unreadable at a distance, mostly because the tip of the up and down arrows have no clear-cut defining line.
Maybe the down arrow could be extended further down to make this distinction more noticeable?
Mockup:
<img width="228" src=https://github.com/user-attachments/assets/629c6b7a-ee66-4ebf-97eb-26702a234810>
Up arrow too, but it's less of a problem.

Let's keep in mind there would be little to no ambiguity with mouse movement, since that cannot be bound, and the actual input's name clears all doubts.

passivestar commented:
> I am a bit iffy on the mouse wheel icons, still. They're an improvement, sure, but are still somewhat unreadable at a distance, mostly because the tip of the up and down arrows have no clear-cut defining line.

Arrow visibility could be improved by ditching the arrow tail and just using a big triangle [like Kenney did](https://kenney.nl/media/pages/assets/input-prompts/adef521913-1744026995/preview-keyboard-mouse.png).

Calinou commented:
> Arrow visibility could be improved by ditching the arrow tail and just using a big triangle [like Kenney did](https://kenney.nl/media/pages/assets/input-prompts/adef521913-1744026995/preview-keyboard-mouse.png).

I've attempted something similar:

<img width="238" height="218" alt="image" src="https://github.com/user-attachments/assets/951dd1ee-cc5d-4845-a7ff-4ad5e5ed1204" />

Unfortunately, the up arrow for the mouse wheel up icon gets cut off.

AdriaandeJongh reviewed (approved):
Tested and this is much better than how it looked previously.

Didn't look at the code.

NicholasBolen commented:
> Arrow visibility could be improved by ditching the arrow tail and just using a big triangle [like Kenney did](https://kenney.nl/media/pages/assets/input-prompts/adef521913-1744026995/preview-keyboard-mouse.png).

Link broke, here's a sample from Kenney's [Input Prompts](https://kenney.nl/assets/input-prompts) (including the outline variant)

<img width="512" height="384" alt="spritesheet" src="https://github.com/user-attachments/assets/37c2c565-e52f-4efd-8e5a-78f75dce41a1" />

Repiteo commented:
Thanks!
